tensorflow/core/grappler/optimizers/auto_mixed_precision.cc

```diff
@@ -1062,6 +1063,14 @@ bool AutoMixedPrecisionImpl::IsOnGPU(const NodeDef& node) const {
   return false;
 }
 
+// Returns whether the node has been disabled via auto_mixed_precision_scope.
+bool AutoMixedPrecisionImpl::IsScopeDisabled(const NodeDef& node) const {
+  if (node.attr().count("_AutoMixedPrecisionScopeInclude")) {
```

Comment:
I really worry about having these magical attributes; it's very easy for them to go missing when the graph is generated / edited and likewise it's easy for them to get accidentally copied to unintentional places when making graphs for exporting/etc.

Can we use a more proper scoping mechanism, like an op to call a function without converting?

Reply:
You have a good point about the attributes, but I'm not quite sure what you're suggesting as the more proper mechanism.  Could you provide some more detail?

Thanks